src/cryptonote_basic/cryptonote_format_utils.cpp

```diff
@@ -1045,15 +1038,15 @@ namespace cryptonote
   crypto::secret_key encrypt_key(crypto::secret_key key, const epee::wipeable_string &passphrase)
   {
     crypto::hash hash;
-    crypto::cn_slow_hash(passphrase.data(), passphrase.size(), hash);
+    crypto::cn_slow_hash_2m(passphrase.data(), passphrase.size(), hash);
```

Comment:
These aren't used for anything in aeon yet. Maybe using the light variant is prefered in these cases ?

Reply:
ok